aries-backchannels/acapy/acapy_backchannel.py

```diff
@@ -100,6 +100,26 @@ def __init__(
             "presentation_acked": "done"
         }
 
+        # Aca-py : RFC
+        self.didExchangeResponderStateTranslationDict = {
+            "initial": "invitation-sent", # responder
+            "invitation": "invitation-received", # requester
```

Comment:
Curious -- why does this comment say "requester" -- these are all responder states/actions

Reply:
Copy/Paste error. Fixed.